CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Main (unreleased)
 
+## v229 (?/??/2021)
+
+* Fix interoperability with other Heroku buildpacks' `$WEB_CONCURRENCY` handling (https://github.com/heroku/heroku-buildpack-ruby/pull/1188)
+
```

Comment:
Also I do think we need a devcenter changelog for this. I pre-write them and store them at this directory: https://github.com/heroku/heroku-buildpack-ruby/tree/main/changelogs. 

Here's an example changelog for this:

```
Ruby buildpack now clears previous default buildpack WEB_CONCURRENCY values

Some buildpacks wills set a default value of the WEB_CONCURRENCY environment variable. For applications using multiple buildpacks, the name of the `.profile.d` script dictates which value takes precedence rather than the order of the buildpack. This can cause problems with the WEB_CONCURRENCY value if a customer has both the `heroku/nodejs` buildpack and the `heroku/ruby` buildpack.

Now if the Ruby buildpack runs after the Node buildpack, it will remove the default value set by nodejs.

Details about the change in behavior can be found on the [pull request to the buildpack](https://github.com/heroku/heroku-buildpack-ruby/pull/1188).
```

Reply:
that's an explanation of the implementation, not of the behavior (which is what matters to customers); I'll write something up ;)